### DISCUSSION TITLE
Install the MrChromebox Legacy Boot Code on Your Chromebook

### DISCUSSION THREAD

**Discussion by CroutonIsFun** [dnschneid/crouton#5069] [General]
Installing the legacy BIOS update requires typing in a special command. The command is tricky, and must be entered correctly. Copy and paste this line into the crosh shell, and press Enter. This will create an alias, allowing you to enter the command correctly in VT-2:

echo 'alias LegacyBoot="cd; curl -LO mrchromebox.tech/firmware-util.sh && sudo bash [firmware-util.sh](https://firmware-util.sh/)"'>>~/.bashrc

After executing the above command, exit the crosh shell with Ctrl-D, and reenter the shell by typing the word "shell" and pressing Enter. This updates your environment.

Now go over to VT-2, login as chronos, and begin typing LegacyBoot. After only 2 or 3 characters, you can press Tab to autocomplete, then press Enter.

Respond to the prompts appropriately, and the legacy code will be uploaded to your Chromebook's BIOS. After creating a USB boot stick with the installer (e.g., Linux Mint), you can boot off the USB by pressing Ctrl-L.

**Comment by DennisLfromGA** [comment]
Thanx but not sure how this pertains to `crouton` ...
Also, not sure why you'd make it an alias and add it to your `.bashrc` since you only need to run it once and you can do it all in VT-2.

**Reply by CroutonIsFun** [reply]
You're right, of course, it does not pertain to crouton. On the other hand, people are looking for alternatives, especially since crouton has been marginalized somewhat. Also, it helps me when I have to reinstall my Chromebook, to have a link in the cloud.

In hopes that this will help others, as well as myself, I went out on a limb, a little. And also, it is the same sort of issue which those who choose to install crouton will face: how to use the crosh shell. Please see the companion article, [5068](5068)

**Comment by CroutonIsFun** [comment]
Will reopen if approriate...
